pipelines/ingest-transforms/src/main/java/org/gbif/pipelines/transforms/core/BasicTransform.java

```diff
@@ -37,7 +38,9 @@
   private final boolean useExtendedRecordId;
   private final BiConsumer<ExtendedRecord, BasicRecord> gbifIdFn;
   private final SerializableSupplier<HBaseLockingKeyService> keygenServiceSupplier;
+  private final SerializableSupplier<VocabularyLookup> lifeStageLookupSupplier;
   private HBaseLockingKeyService keygenService;
+  private VocabularyLookup lifeStageLookup;
```

Comment:
Let's wait to see what ALA would need. Because if they need to customize all this maybe it doesn't make sense for them to use our vocabularies at all. The vocabularies are supposed to be generic, at this point I think the only thing that can be different for them is how to create the `VocabularyLookup` instances (reading from a file, downloading from the WS, etc.).